<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="6" passed="3" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-05-10T04:57:37Z" name="Suite" finished-at="2021-05-10T04:58:14Z" duration-ms="36981">
    <groups>
    </groups>
    <test started-at="2021-05-10T04:57:37Z" name="Test" finished-at="2021-05-10T04:58:14Z" duration-ms="36981">
      <class name="myTests.baseTestClass">
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:38Z" name="sample" finished-at="2021-05-10T00:57:41Z" duration-ms="3028" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="InValidLogin(java.lang.String, java.lang.String)[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:41Z" name="InValidLogin" data-provider="inValidLoginCredentials" finished-at="2021-05-10T00:57:42Z" duration-ms="995" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Test13232]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[MUCT-air0tuss3sars]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Message Provided not matched with the Actual Result expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Message Provided not matched with the Actual Result expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.mypages.base.verifyMessagePrompted(base.java:83)
	at myTests.baseTestClass.InValidLogin(baseTestClass.java:89)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InValidLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:42Z" name="tearDown" finished-at="2021-05-10T00:57:43Z" duration-ms="759" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:43Z" name="sample" finished-at="2021-05-10T00:57:45Z" duration-ms="2387" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="ValidLogin(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:45Z" name="ValidLogin" data-provider="ValidLoginCredentialsWithKeyWordSearch" finished-at="2021-05-10T00:57:56Z" duration-ms="10967" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[qecandidate]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[MUCT-air0tuss3sars]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[mabl]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:57Z" name="tearDown" finished-at="2021-05-10T00:57:57Z" duration-ms="657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:57:57Z" name="sample" finished-at="2021-05-10T00:58:00Z" duration-ms="2613" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="VerifyElementsPresentInLoginPage()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:00Z" name="VerifyElementsPresentInLoginPage" finished-at="2021-05-10T00:58:00Z" duration-ms="221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyElementsPresentInLoginPage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:00Z" name="tearDown" finished-at="2021-05-10T00:58:01Z" duration-ms="649" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:01Z" name="sample" finished-at="2021-05-10T00:58:03Z" duration-ms="2573" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="VerifyLoginWithJustPassword(java.lang.String, java.lang.String)[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:03Z" name="VerifyLoginWithJustPassword" data-provider="inValidLoginCredentials" finished-at="2021-05-10T00:58:04Z" duration-ms="564" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Test13232]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[MUCT-air0tuss3sars]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Message Provided not matched with the Actual Result expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Message Provided not matched with the Actual Result expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.mypages.base.verifyMessagePrompted(base.java:83)
	at myTests.baseTestClass.VerifyLoginWithJustPassword(baseTestClass.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLoginWithJustPassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:04Z" name="tearDown" finished-at="2021-05-10T00:58:05Z" duration-ms="719" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:05Z" name="sample" finished-at="2021-05-10T00:58:07Z" duration-ms="2454" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="VerifyLoginWithJustUserName(java.lang.String, java.lang.String)[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:07Z" name="VerifyLoginWithJustUserName" data-provider="inValidLoginCredentials" finished-at="2021-05-10T00:58:08Z" duration-ms="778" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Test13232]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[MUCT-air0tuss3sars]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Message Provided not matched with the Actual Result expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Message Provided not matched with the Actual Result expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.mypages.base.verifyMessagePrompted(base.java:83)
	at myTests.baseTestClass.VerifyLoginWithJustUserName(baseTestClass.java:95)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLoginWithJustUserName -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:08Z" name="tearDown" finished-at="2021-05-10T00:58:09Z" duration-ms="646" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="sample()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:09Z" name="sample" finished-at="2021-05-10T00:58:11Z" duration-ms="2336" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
        <test-method signature="verifySearchResults(java.lang.String)[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:11Z" name="verifySearchResults" finished-at="2021-05-10T00:58:14Z" duration-ms="2362" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mabl]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySearchResults -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:myTests.baseTestClass@18025ced]" started-at="2021-05-10T00:58:14Z" name="tearDown" finished-at="2021-05-10T00:58:14Z" duration-ms="667" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- myTests.baseTestClass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
